plugins/destination/file/client/client.go

```diff
@@ -18,23 +18,21 @@ type Client struct {
 	spec       specs.Destination
 	pluginSpec Spec
 
-	csvTransformer         *csv.Transformer
-	csvReverseTransformer  *csv.ReverseTransformer
-	jsonTransformer        *json.Transformer
-	jsonReverseTransformer *json.ReverseTransformer
+	CSVClient              *csv.Client
+	CSVTransformer         csv.Transformer
+	CSVReverseTransformer  csv.ReverseTransformer
+	JSONClient             *json.Client
+	JSONTransformer        json.Transformer
+	JSONReverseTransformer json.ReverseTransformer
```

Comment:
Is there a reason why you needed to change the transformers?

Reply:
Nice catch, no need to do it, left over from the old PR. Fixed in https://github.com/cloudquery/cloudquery/pull/7466/commits/2b3a8beabfdd5b780fd686b78ef46cdb8e23e759